src/sql/src/statements/transform/type_alias.rs

```diff
@@ -384,13 +397,13 @@ CREATE TABLE data_types (
   tt TEXT,
   mt TEXT,
   lt TEXT,
-  tint INT8,
+  tint TINYINT,
   sint SMALLINT,
   i INT,
-  bint INT64,
+  bint BIGINT,
   v VARCHAR,
   f FLOAT,
-  d FLOAT64,
+  d DOUBLE,
```

Comment:
Not sure if we have some document about these that need to be updated

Reply:
Maybe we don't have to do that. This is the expected behavior implemented in https://github.com/GreptimeTeam/greptimedb/pull/2331

We forgot to fix this when we updated the sqlparser. You can find out some types have alias but some do not in this test. Now this PR fixes this and removes the TODO. cc @killme2008